compiler/mx.compiler/mx_compiler.py

```diff
@@ -962,6 +962,11 @@ def sl(args):
     mx.get_opts().jdk = 'jvmci'
     mx_truffle.sl(args)
 
+def lib_folder():
```

Comment:
```suggestion
def jdk_lib_folder():
```

This is a slightly more precise name.